### PR TITLE
Bump Traefik to v2.0.0-rc2

### PR DESCRIPTION
**Pull request by ldez**
Hello,

This PR updates Traefik to v2.0.0-rc2.

/cc @mmatur @emilevauge @juliens 

Cheers

### CODE REVIEW

**Comment by juliens** [comment]
LGTM

File: library/traefik [hunk 1]
```diff
@@ -1,19 +1,19 @@
 Maintainers: Emile Vauge <emile@vauge.com> (@emilevauge), Vincent Demeester <vincent@sbr.pm> (@vdemeester), Ludovic Fernandez <ludovic@containo.us> (@ldez), Julien Salleyron <julien@containo.us> (@juliens), Nicolas Mengin <nicolas@containo.us> (@nmengin), Michael Matur <michael@containo.us> (@mmatur), Damien Duportal <damien@containo.us> (@dduportal)
 GitRepo: https://github.com/containous/traefik-library-image.git
 
-Tags: v2.0.0-rc1, 2.0.0-rc1, v2.0, 2.0, montdor
+Tags: v2.0.0-rc2, 2.0.0-rc1, v2.0, 2.0, montdor
```
Comment (LeoColomb) [inline library/traefik]:
```suggestion
Tags: v2.0.0-rc2, 2.0.0-rc2, v2.0, 2.0, montdor
```

?

If so, all tags lines are affected.

**Comment by tianon** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index b9481da..8f72ae4 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -10,10 +10,10 @@ traefik:2.0
 traefik:2.0-alpine
 traefik:2.0-nanoserver
 traefik:2.0-nanoserver-sac2016
-traefik:2.0.0-rc1
-traefik:2.0.0-rc1-alpine
-traefik:2.0.0-rc1-nanoserver
-traefik:2.0.0-rc1-nanoserver-sac2016
+traefik:2.0.0-rc2
+traefik:2.0.0-rc2-alpine
+traefik:2.0.0-rc2-nanoserver
+traefik:2.0.0-rc2-nanoserver-sac2016
 traefik:alpine
 traefik:latest
 traefik:maroilles
@@ -38,7 +38,7 @@ traefik:v2.0
 traefik:v2.0-alpine
 traefik:v2.0-nanoserver
 traefik:v2.0-nanoserver-sac2016
-traefik:v2.0.0-rc1
-traefik:v2.0.0-rc1-alpine
-traefik:v2.0.0-rc1-nanoserver
-traefik:v2.0.0-rc1-nanoserver-sac2016
+traefik:v2.0.0-rc2
+traefik:v2.0.0-rc2-alpine
+traefik:v2.0.0-rc2-nanoserver
+traefik:v2.0.0-rc2-nanoserver-sac2016
diff --git a/traefik_montdor-alpine/Dockerfile b/traefik_montdor-alpine/Dockerfile
index adf5e30..e75e717 100644
--- a/traefik_montdor-alpine/Dockerfile
+++ b/traefik_montdor-alpine/Dockerfile
@@ -8,7 +8,7 @@ RUN set -ex; \
 		x86_64) arch='amd64' ;; \
 		*) echo >&2 "error: unsupported architecture: $apkArch"; exit 1 ;; \
 	esac; \
-	wget --quiet -O /tmp/traefik.tar.gz "https://github.com/containous/traefik/releases/download/v2.0.0-rc1/traefik_v2.0.0-rc1_linux_$arch.tar.gz"; \
+	wget --quiet -O /tmp/traefik.tar.gz "https://github.com/containous/traefik/releases/download/v2.0.0-rc2/traefik_v2.0.0-rc2_linux_$arch.tar.gz"; \
 	tar xzvf /tmp/traefik.tar.gz -C /usr/local/bin traefik; \
 	rm -f /tmp/traefik.tar.gz; \
 	chmod +x /usr/local/bin/traefik
@@ -22,5 +22,5 @@ LABEL org.opencontainers.image.vendor="Containous" \
 	org.opencontainers.image.url="https://traefik.io" \
 	org.opencontainers.image.title="Traefik" \
 	org.opencontainers.image.description="A modern reverse-proxy" \
-	org.opencontainers.image.version="v2.0.0-rc1" \
+	org.opencontainers.image.version="v2.0.0-rc2" \
 	org.opencontainers.image.documentation="https://docs.traefik.io"
diff --git a/traefik_montdor-nanoserver-sac2016/Dockerfile b/traefik_montdor-nanoserver-sac2016/Dockerfile
index 19cbc70..d974f23 100644
--- a/traefik_montdor-nanoserver-sac2016/Dockerfile
+++ b/traefik_montdor-nanoserver-sac2016/Dockerfile
@@ -1,7 +1,7 @@
 FROM microsoft/nanoserver:sac2016
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 RUN Invoke-WebRequest \
-        -Uri "https://github.com/containous/traefik/releases/download/v2.0.0-rc1/traefik_v2.0.0-rc1_windows_amd64.zip" \
+        -Uri "https://github.com/containous/traefik/releases/download/v2.0.0-rc2/traefik_v2.0.0-rc2_windows_amd64.zip" \
         -OutFile "/traefik.zip"  ; \
     Expand-Archive -Path "/traefik.zip" -DestinationPath "/" -Force  ; \
     Remove-Item –path /traefik.zip
@@ -14,5 +14,5 @@ LABEL org.opencontainers.image.vendor="Containous" \
     org.opencontainers.image.url="https://traefik.io" \
     org.opencontainers.image.title="Traefik" \
     org.opencontainers.image.description="A modern reverse-proxy" \
-    org.opencontainers.image.version="v2.0.0-rc1" \
+    org.opencontainers.image.version="v2.0.0-rc2" \
     org.opencontainers.image.documentation="https://docs.traefik.io"
diff --git a/traefik_montdor/Dockerfile b/traefik_montdor/Dockerfile
index d66487a..f0e25d3 100644
--- a/traefik_montdor/Dockerfile
+++ b/traefik_montdor/Dockerfile
@@ -1,7 +1,7 @@
 FROM scratch
-COPY --from=traefik:2.0.0-rc1-alpine /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
-COPY --from=traefik:2.0.0-rc1-alpine /usr/share/zoneinfo /usr/share/
-COPY --from=traefik:2.0.0-rc1-alpine /usr/local/bin/traefik /
+COPY --from=traefik:v2.0.0-rc2-alpine /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=traefik:v2.0.0-rc2-alpine /usr/share/zoneinfo /usr/share/
+COPY --from=traefik:v2.0.0-rc2-alpine /usr/local/bin/traefik /
 
 EXPOSE 80
 VOLUME ["/tmp"]
@@ -12,5 +12,5 @@ LABEL org.opencontainers.image.vendor="Containous" \
 	org.opencontainers.image.url="https://traefik.io" \
 	org.opencontainers.image.title="Traefik" \
 	org.opencontainers.image.description="A modern reverse-proxy" \
-	org.opencontainers.image.version="v2.0.0-rc1" \
+	org.opencontainers.image.version="v2.0.0-rc2" \
 	org.opencontainers.image.documentation="https://docs.traefik.io"
````

</details>

**Comment by tianon** [comment]
FYI, `microsoft/nanoserver:sac2016` is deprecated and hasn't been receiving updates for a while now.

**Comment by tianon** [comment]
LGTM

Build test of #6574; 52624551; `amd64` (`traefik`):

```console

$ bashbrew build traefik:v2.0.0-rc2-alpine
Building bashbrew/cache:2b9163eba47c0b5934d0aff073a59e2f9bdbc917ed98965910e06597833983d6 (traefik:v2.0.0-rc2-alpine)
Tagging traefik:v2.0.0-rc2-alpine
Tagging traefik:2.0.0-rc2-alpine
Tagging traefik:v2.0-alpine
Tagging traefik:2.0-alpine
Tagging traefik:montdor-alpine

$ test/run.sh traefik:v2.0.0-rc2-alpine
testing traefik:v2.0.0-rc2-alpine
	'cve-2014--shellshock' [1/2]...passed
	'override-cmd' [2/2]...= 'echo' is not a Traefik command: assuming shell execution.
passed


$ bashbrew build traefik:v2.0.0-rc2-nanoserver
skipping "traefik:v2.0.0-rc2-nanoserver" (due to architecture "amd64"; only "windows-amd64" supported)

$ test/run.sh traefik:v2.0.0-rc2-nanoserver
testing traefik:v2.0.0-rc2-nanoserver
	image has no tests...skipping


$ bashbrew build traefik:v1.7.14
Using bashbrew/cache:79ee3af0626f04f7e60b364fa4a0c39f447366f19d42fac548c62cfc911bf9e8 (traefik:v1.7.14)
Tagging traefik:v1.7.14
Tagging traefik:1.7.14
Tagging traefik:v1.7
Tagging traefik:1.7
Tagging traefik:maroilles
Tagging traefik:latest

$ test/run.sh traefik:v1.7.14
testing traefik:v1.7.14
	'cve-2014--shellshock' [1/2]...passed
	'override-cmd' [2/2]...image does not appear to contain "echo" -- assuming single-binary image
passed


$ bashbrew build traefik:v1.7.14-alpine
Using bashbrew/cache:0a96f3735cfa0b8f11998939087b694a0a89e120a3980db1288a82dd07284738 (traefik:v1.7.14-alpine)
Tagging traefik:v1.7.14-alpine
Tagging traefik:1.7.14-alpine
Tagging traefik:v1.7-alpine
Tagging traefik:1.7-alpine
Tagging traefik:maroilles-alpine
Tagging traefik:alpine

$ test/run.sh traefik:v1.7.14-alpine
testing traefik:v1.7.14-alpine
	'cve-2014--shellshock' [1/2]...passed
	'override-cmd' [2/2]...= 'echo' is not a Traefik command: assuming shell execution.
passed


$ bashbrew build traefik:v1.7.14-nanoserver
skipping "traefik:v1.7.14-nanoserver" (due to architecture "amd64"; only "windows-amd64" supported)

$ test/run.sh traefik:v1.7.14-nanoserver
testing traefik:v1.7.14-nanoserver
	image has no tests...skipping


$ bashbrew build traefik:v2.0.0-rc2
Building bashbrew/cache:a67968707ae7f4a1b8d198909b885259616e13999319a7da7519e3bc63243776 (traefik:v2.0.0-rc2)
Tagging traefik:v2.0.0-rc2
Tagging traefik:2.0.0-rc2
Tagging traefik:v2.0
Tagging traefik:2.0
Tagging traefik:montdor

$ test/run.sh traefik:v2.0.0-rc2
testing traefik:v2.0.0-rc2
	'cve-2014--shellshock' [1/2]...passed
	'override-cmd' [2/2]...image does not appear to contain "echo" -- assuming single-binary image
passed

```

**Pull request merged**